tests/dashbio_demos/app_sequence_viewer.py

```diff
@@ -69,8 +69,8 @@ def description():
 
 def layout():
 
-    return html.Div(id='seq-view-body', className='app-body', children=[
-        dcc.Loading(className='dashbio-loading', children=html.Div(
+    return html.Div(id='seq-view-body', className='app-body dashbio-loading', children=[
```

Comment:
```suggestion
    return html.Div(id='seq-view-body', className='app-body', children=[
```

We don't need the loading state class since we're getting rid of the `dcc.Loading` component!